website_sale_delivery_date/static/src/js/checkout_delivery_date.esm.js

```
@@ -0,0 +1,83 @@
+                rounding: 30,
+                value: parseDateTime(value),
+            },
+        });
+        await this.picker.enable();
+    },
+
+    async _onChangeDatePicker(newDate) {
+        const carrierEl = this.el.querySelector(
+            'input[name="o_delivery_radio"]:checked'
+        );
+        const carrierId = parseInt(carrierEl.dataset.dmId, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)